patches/chrome-tools-build-mac-infoplist_strings_util.mm.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/chrome/tools/build/mac/infoplist_strings_util.mm b/chrome/tools/build/mac/infoplist_strings_util.mm
+index f1d158dca8b844a73919d28e754c6f45cb0a9c81..7a99b494e9fd3a7be8ece3a6cd97011b819f4b27 100644
+--- a/chrome/tools/build/mac/infoplist_strings_util.mm
++++ b/chrome/tools/build/mac/infoplist_strings_util.mm
+@@ -247,8 +247,7 @@ int main(int argc, char* const argv[]) {
+ 
+     // Fetch the strings
+     NSString* name =
+-          LoadStringFromDataPack(branded_data_pack.get(), cur_lang,
```

Comment:
this doesn't seem right to me because presumably this works with multi-channel chrome